### PR TITLE
gitlab-ci-multi-runner 1.9.2

### PR DESCRIPTION
**Pull request by JCount**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
also add ARM prebuilt image,
static build time

### CODE REVIEW

**Comment by ilovezfs** [comment]
@tmaczukin mind verifying this is [as you requested](https://github.com/Homebrew/homebrew-core/pull/8148#issuecomment-270194212)?

File: Formula/gitlab-ci-multi-runner.rb [hunk 1]
```diff
@@ -41,21 +48,24 @@ def install
     end
 
     cd dir do
-      resource("prebuilt-x86_64.tar.xz").stage do
-        system "go-bindata", "-pkg", "docker", "-nocompress", "-nomemcopy",
-                             "-nometadata", "-o",
-                             "#{dir}/executors/docker/bindata.go",
-                             "prebuilt-x86_64.tar.xz"
+      ["prebuilt-x86_64.tar.xz", "prebuilt-arm.tar.xz"].each do |r|
```
Comment (tmaczukin) [inline Formula/gitlab-ci-multi-runner.rb]: I don't know how Homebrew works internally, so maybe my comment is invalid, but from what I see here it looks that the `go-bindata -pkg ...` command will be executed twice: first time with `prebuilt-x86_64.tar.xz` as the argument and second time with `prebuilt-arm.tar.xz` as the argument. If yes, then the second execution will overwrite the `bindata.go` file. And that's not the goal which we want to achieve.

It should be a one execution of this command with both files as the final arguments, so something looking like:
```
go-bindata -pkg docker \
           -nocompress \
           -nomemcopy \
           -nometadata \
           -o executors/docker/bindata.go
           prebuilt-x86_64.tar.xz
           prebuilt-arm.tar.xz
```

With this both files will be available for Docker executor as assets stored internally in `bindata.go` file.

**Comment by tmaczukin** [comment]
@ilovezfs Thanks for mentioning me. Generally it looks good. However I left one comment in a place that worries me. If this will be fixed or confirmed that the current implementation works as it should, then I think it can be merged.

**Comment by ilovezfs** [comment]
@tmaczukin should be fixed now

**Comment by tmaczukin** [comment]
Looks good for me :)

**Pull request closed without merging**